specs/_features/eip7732/p2p-interface.md

```diff
@@ -215,9 +216,44 @@ _ _[IGNORE]_ `header.parent_block_root` is the hash tree root of a known beacon
 | `DENEB_FORK_VERSION`     | `deneb.SignedBeaconBlock`     |
 | `EIP7732_FORK_VERSION`   | `eip7732.SignedBeaconBlock`   |
 
-##### BeaconBlocksByRoot v3
+##### ExecutionPayloadEnvelopesByRange v1
 
-**Protocol ID:** `/eth2/beacon_chain/req/beacon_blocks_by_root/3/`
+**Protocol ID:** `/eth2/beacon_chain/req/execution_payload_envelopes_by_range/1/`
+
+*[New in EIP-7732]*
+
+Request Content:
+
+```
+(
+  start_slot: Slot
+  count: uint64
+)
+```
+
+Response Content:
+
+```
+(
+  List[SignedExecutionPayloadEnvelope, MAX_REQUEST_BLOCKS_DENEB]
+)
+```
+
+Specifications of req\response method are equivalent to [BeaconBlocksByRoot v2](#beaconblocksbyroot-v2), with the only difference that response content.
+
+For each `response_chunk`, a `ForkDigest`-context based on `compute_fork_version(compute_epoch_at_slot(signed_execution_payload_envelop.message.slot))` is used to select the fork namespace of the Response type.
+
+Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
+
+[0]: # (eth2spec: skip)
+
+| `fork_version`         | Chunk SSZ type                           |
+|------------------------|------------------------------------------|
+| `EIP7732_FORK_VERSION` and later | `eip7732.SignedExecutionPayloadEnvelope` |
```

Comment:
I don't think we should use "and later" here because there might be new `SignedExecutionPayloadEnvelope` containers in the future. If/when we update `ExecutionPayload` or `ExecutionRequests` there would need to be a new `fork.SignedExecutionPayloadEnvelope` container.